clients/java/src/test/java/io/zeebe/client/ZeebeClientTest.java

```diff
@@ -187,4 +188,24 @@ public void shouldBrokerContactPointReturnTheSameAsGatewayAddress() {
           .isEqualTo(gatewayAddress);
     }
   }
+
+  @Test
+  public void shouldCloudBuilderBuildProperClient() {
```

Comment:
Considering the logic is now different for construction of the cloud client. Please add some test cases to verify the possibility to overwrite the gatewayAddress and the credentialsProvider.

Reply:
Done